src/pages/BitcoinDashboardV2/components/PendingWithdraws/PendingWithdraws.tsx

```
@@ -0,0 +1,127 @@
+    if (baseToken) BaseTokenIcon = theme === 'light' ? baseToken.Light : baseToken.Dark;
+    if (quoteToken) QuoteTokenIcon = theme === 'light' ? quoteToken.Light : quoteToken.Dark;
+
+    return (
+      <div className={styles.symbols}>
+        <BaseTokenIcon className={styles.symbols_logo_left} />
+        <QuoteTokenIcon className={styles.symbols_logo_right} />
+      </div>
+    );
+  };
+
+  const handleNavigate = (txid: String) => {
```

Biome

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)